Block/Product/View.php

```diff
@@ -210,6 +210,7 @@ protected function _toHtml()
         if ($this->getBlockCode() === \Bread\BreadCheckout\Helper\Data::BLOCK_CODE_PRODUCT_VIEW
             && $this->catalogHelper->isEnabledOnPDP()
             && $this->catalogHelper->allowedProductType($this->getProduct()->getTypeId())
+            && $this->dataHelper->aboveThreshold($this->getProduct()->getPriceInfo()->getPrice('final_price')->getValue())
```

Comment:
same thing with the breaking this out into a variable like you did elsewhere

Reply:
No, this needs to be this way